### PR TITLE
7z, 7za, 7zr: fix Japanese translation

### PR DESCRIPTION
**Pull request by Ry0taK**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Japanese translation of 7z were bit broken (e.g. `道/に/ファイルまたはディレクトリ` means `File or directory on the road`), so I tweaked it a bit.

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]
Look good to me. Thanks for contribution.

File: pages.ja/common/7z.md [hunk 1]
```diff
@@ -1,36 +1,36 @@
 # 7z
 
-> 7-Zipは高圧縮率のファイルアーカイバ
-> 詳しくはこちら: <https://www.7-zip.org>.
+> 圧縮率の高いファイルアーカイバです。
+> 詳しくはこちら: <https://www.7-zip.org>
```
Comment (navarroaxel) [inline pages.ja/common/7z.md:4]: We need a period here
```suggestion
> 詳しくはこちら: <https://www.7-zip.org>.
```

Comment (Ry0taK) [reply pages.ja/common/7z.md:4]: Thank you for pointing it! Is this period necessary for TLDR to parse markdown correctly?

Comment (navarroaxel) [reply pages.ja/common/7z.md:4]: It's not required for the markdown, but it's required to keep a format without lints. We don't have a lint for ja so we should do this manually.

**Review by navarroaxel — APPROVED** [review]

File: pages.ja/common/7z.md (hunk 1 above)
Comment (Ry0taK) [reply pages.ja/common/7z.md]: I understand it, I've added period to Japanese translation of `7z*`

**Pull request merged**